static/style.css

```diff
@@ -19,11 +20,26 @@
 
 @media only screen and (max-width: 1160px) {
     .content {
-		width: 100%;
+		width: 98%;
 		margin: 50px 0 80px 0;
     }
 }
 
+.contentReg {
+	position: absolute;
+	left: 35%;
+	/*height: 65%;*/
+	width: 30%;
+	margin: 4% 0 6% 0;
+	background-color: #F4F4F4;
+    padding: 0;
```

Comment:
Same as above and throughout the rest of the file, stay with one indentation type.

Reply:
5a10b9a